atomic_reactor/inner.py

```diff
@@ -483,7 +485,7 @@ def build_docker_image(self):
         :return: BuildResult
         """
         exception_being_handled = False
-        self.builder = InsideBuilder(self.source, self.image)
+        self.builder = InsideBuilder(self.source, self.user_params['image_tag'])
```

Comment:
why not use already that image property? 
meaning you can leave there : self.image